settings.gradle

```diff
@@ -8,3 +8,4 @@ include 'alerting'
 include 'core'
 
 project(":core").name = 'alerting-core'
+startParameter.excludedTaskNames=["validatePluginZipPom"]
```

Comment:
Since you are configuring the POM above you dont need to exclude this `validatePluginZipPom`.

Reply:
Removed